testutil/promrated/promrated.go

```diff
@@ -42,7 +43,7 @@ func Run(ctx context.Context, config Config) error {
 		serverErr <- serveMonitoring(config.MonitoringAddr, promRegistry)
 	}()
 
-	ticker := time.NewTicker(12 * time.Hour)
+	ticker := time.NewTicker(24 * time.Hour)
```

Comment:
What's the reason behind updating this ticker? to reduce number of queries per day? A comment would be great for the same.